Mathlib/Data/Vector.lean

```diff
@@ -13,6 +13,7 @@ import Std.Data.List.Basic
 import Std.Data.List.Lemmas
 import Mathlib.Init.Data.List.Basic
 import Mathlib.Init.Data.List.Lemmas
+import Mathlib.Data.Nat.Pow
```

Comment:
Is this actually needed?

Reply:
I don't think definitions from `Data.Nat.Pow` itself are needed, but it does transitively import a bunch of files that are needed, and I couldn't find which files these are.
Specifically, I couldn't find where an instance for `ExistsAddOfLE ℕ` is defined, but importing that file seems to give it.